mst-content-hint/idlharness.window.js

```diff
@@ -0,0 +1,19 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+// META: script=/webrtc/RTCPeerConnection-helper.js
+
+'use strict';
+
+idl_test(
+  ['mst-content-hint'],
+  ['mediacapture-streams', 'dom'],
+  async idl_array => {
+    idl_array.add_objects({
+      MediaStreamTrack: ['audioTrack', 'videoTrack'],
+    });
+
+    const stream = await getNoiseStream({ audio: true, video: true });
+    self.audioTrack = stream.getAudioTracks()[0];
+    self.videoTrack = stream.getAudioTracks()[0];
```

Comment:
The above nit assumes it was intended to re-use the same value. Is it? Or is there a getVideoTracks equivalent?

Reply:
Oops, copypasta, will fix.